build.gradle

```diff
@@ -3,11 +3,11 @@ buildscript {
         buildConfig = [
                 'compileSdk': 27,
                 'minSdk': 19,
-                'buildTools': '27.0.3'
+                'buildTools': '28.0.3'
```

Comment:
Please also change `.travis.yml` to install the new build tools version. That should fix the CI build in this pull request.